debug.go

```diff
@@ -2,17 +2,16 @@ package whisper
 
 import (
 	"fmt"
-	"os"
+	"math"
 	"time"
 )
 
-func (whisper *Whisper) CheckIntegrity() {
+func (whisper *Whisper) CheckIntegrity() (msg string, good bool) {
```

Comment:
why not just return `error` and do typical `if err != nil` check in the main body?

Reply:
It's a good point. error is better.

Reply:
Fixed.